LDAR_Sim/external_sensors/METEC_NO_WIND.py

```diff
@@ -0,0 +1,149 @@
+        covered_equipment_rates (list): list of equipment leak rates that can be
+                                        detected by the crew
+        covered_site_rate (float): total site emissions from leaks that are observable
+                                    from a crew
+        site_rate (float): total site emissions from leaks all leaks at site
+        venting (float): total site emissions from venting
+        equipment_rates (list): list of equipment leak rates for each equipment group
+
+    Returns:
+        site report (dict):
+                site  (site obj): same as input
+                leaks_present (list): same as covered leaks input
+                site_true_rate (float): same as site_rate
+                site_measured_rate (float): total emis from all leaks measured
+                equip_measured_rates (list): total of all leaks measured for each equip group
+                venting (float): same as input
+                found_leak (boolean): Did the crew find at least one leak at the site
+
+
+    """
+
+    missed_leaks_str = '{}_missed_leaks'.format(self.config['label'])
+    equip_measured_rates = []
+    site_measured_rate = 0
+    found_leak = False
+    n_leaks = len(covered_leaks)
+    mdl = self.config['sensor']['MDL']
+
+    if self.config["measurement_scale"] == "site":
+        # factor of 3.6 converts g/s to kg/h
+        rate = covered_site_rate * 3.6
+        prob_detect = 1/(1+np.exp(mdl[0]-mdl[1]*rate))
+        if rate <= (mdl[2]*3.6):
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+        elif np.random.binomial(1, prob_detect):
+            found_leak = True
+            site_measured_rate = measured_rate(
+                covered_site_rate, self.config['sensor']['QE'])
+        else:
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+    elif self.config["measurement_scale"] == "equipment":
+        for rate in covered_equipment_rates:
+            m_rate = measured_rate(rate, self.config['sensor']['QE'])
+            rate = m_rate * 3.6
+            prob_detect = 1 / \
+                (1+np.exp(mdl[0]-mdl[1]*rate))
+            if rate <= (mdl[2]*3.6):
+                m_rate = 0
+            elif np.random.binomial(1, prob_detect):
+                found_leak = True
+            else:
+                m_rate = 0
+            equip_measured_rates.append(m_rate)
+            site_measured_rate += m_rate
+        if not found_leak:
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+    elif self.config['measurement_scale'] == 'component':
+        for leak in covered_leaks:
+            rate = leak['rate'] * 3.6
+            prob_detect = 1 / \
+                (1+np.exp(mdl[0]-mdl[1]*rate))
+            if prob_detect >= 1:
```

Comment:
Why is this logic only needed in the case of "component" level detection?

Reply:
Should be redundant, however just to ensure that if someone does provides coefficients that result in too big of number that could cause issues, the check will be added into the other levels.